data/rule_data.yml

```diff
@@ -98,6 +98,13 @@ rule_data:
   - name: io.openshift.tags
   - name: summary
 
+  # https://enterprisecontract.dev/docs/ec-policies/release_policy.html#labels__optional_disallowed_inherited_labels
```

Comment:
This link doesn't exist yet because the website build is currently broken,  [EC-396](https://issues.redhat.com/browse/EC-396).